manifests/bird_exporter.pp

```diff
@@ -47,6 +47,8 @@
 #  User which runs the service
 # @param version
 #  The binary release version
+# @param env_vars
+#  Optional hash with environment variables thats passed to the exporter via init script / unit file
```

Comment:
Probably lose the word 'Optional'.  It implies the `Optional` type.

Reply:
fixed it